draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -675,6 +725,10 @@ Routers SHOULD whenever possible, split the information between the fewest possi
 </list>
 </t>
 
+<t>When a router distributes information across multiple Router Advertisements (RAs), it SHOULD minimize the time interval between them.
```

Comment:
I believe this should spell out what this means.

i.e., whether all these RAs simply count as one (and you'd then get a higher overall RA rate), or whether you'd keep the same overall RA rate -- but then  the RA rate for each prefix would reduced?

Me, I;d go with the former... because otherwise that would mean that as the number of PIOs increases, you'd risk not refreshing them....   (since you send one RA per PIO, and you want to keep the overall RA rate constant, then you ahve to reduce the RA-rate for each PIO...)

Reply:
Ideally the former, yes, but I'm not sure we can do both:ask the router to maintain separate timers for each RA **AND** send them together in a bundle..

Reply:
WHat I believe is that there should be one recommendation regarding what to do (at least "unless they know better"). My take would be to go for the simpler/safest option, and consider that all of these RAs count as a single one.   (this won't risk prefixes failing to be refreshed... although comes at a price of possible higher RA rate).

Reply:
I've updated the text, PTAL